tools/SourceKit/lib/SwiftLang/SwiftASTManager.cpp

```diff
@@ -888,35 +876,27 @@ ASTBuildOperation::fileContentsAndStampsForFilesInCompilerInvocation() {
       Content->Buffer =
           llvm::WritableMemoryBuffer::getNewMemBuffer(0, Filename);
     }
-    Stamps.push_back(Content->Stamp);
     FileContents.push_back(Content);
   }
-  assert(Stamps.size() ==
-         Opts.Invok.getFrontendOptions().InputsAndOutputs.inputCount());
   assert(FileContents.size() ==
          Opts.Invok.getFrontendOptions().InputsAndOutputs.inputCount());
-  return std::make_pair(FileContents, Stamps);
+  return FileContents;
 }
 
 bool ASTBuildOperation::matchesSourceState(
     llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> OtherFileSystem) {
   const InvocationOptions &Opts = InvokRef->Impl.Opts;
 
-  // Check if the inputs changed.
-  std::vector<BufferStamp> InputStamps;
-  InputStamps.reserve(
-      Opts.Invok.getFrontendOptions().InputsAndOutputs.inputCount());
   // IMPORTANT: The computation of stamps must match the one in
   // snapshotAndStampsForFilesInCompilerInvocation.
```

Comment:
This function doesn't exist any more

Reply:
Removed the comment. It was also out of date.